### DISCUSSION TITLE
techdoc not rendering the top level menus

### DISCUSSION THREAD

**Discussion by Keerthanalaxmi** [squidfunk/mkdocs-material#7412] [Q&A] [answered]
Hi All. I have a mkdocs.yaml file and it has top level menus & side bar. While building the documentation for UI, it appears properly. When I register the catalog.yaml in backstage, the top level menus & side bar are appearing in sidebar. Any idea why would I get in to this issue?

**Comment by squidfunk** [comment, marked as answer]
Thanks for asking. We're not the maintainers of Backstage/Techdocs. This is Material for MkDocs, which is used by [Backstage](https://github.com/backstage/backstage). Thus, please report/ask this upstream ☺️
